server/services/bookingService.ts

```diff
@@ -21,6 +28,18 @@ export default class BookingService {
     return confirmedBooking
   }
 
+  async createForBedspace(token: string, premisesId: string, room: Room, booking: NewBooking): Promise<Booking> {
+    const bookingClient = this.bookingClientFactory(token)
+
+    const confirmedBooking = await bookingClient.create(premisesId, {
+      serviceName: 'temporary-accommodation',
+      bedId: room.beds[0].id,
```

Comment:
What happens if a room doesn't have any beds?

Reply:
I feel ok with this just erroring. All TA rooms should have a bed, and if not, something has gone wrong somewhere. In the case that a room doesn't have a bed, this isn't something we can elegantly handle I feel ok with that erroring here.

Reply:
👍